taskwiki/vwtask.py

```diff
@@ -346,13 +346,17 @@ def find_parent_task(self):
             return None
 
         for i in reversed(range(0, self['line_number'])):
-            # Stop looking if there is less indentation
-            indentation = len(self.cache.buffer[i]) - len(self.cache.buffer[i].lstrip())
+            # Tab is equal to four spaces
+            # TODO: Source this from shiftwidth setting, but this would be only
```

Comment:
`tabstop` rather than `shiftwidth`?

`sw` doesn't affect display at all, it only affects the behaviour of <kbd>Tab</kbd> when editing.

Reply:
Good catch, it was some time since I setup these settings in my dotfiles :slightly_smiling_face: